gui-agent/vmside.c

```diff
@@ -2188,6 +2188,12 @@ static void handle_sigterm()
     exit(0);
 }
 
+static void handle_sigchld()
```

Comment:
```suggestion
static void handle_sigchld(void)
```

Otherwise recent clang will (rightly) complain.  That said, one should really use the `siginfo_t` argument to check that it was in fact Xorg that exited, as opposed to e.g. a signal sent from another process.  Also, Xorg’s exit code should be logged.